test/unit/vl-wizard.test.html

```diff
@@ -147,30 +147,45 @@ <h3 slot="title">Stap 3</h3>
         wizard._panes.forEach((pane) => assert(pane.enablePreviousPane.called));
       });
 
+      test('via een attribuut zal de navigatie van met vertraging toegevoegde stappen uitgeschakeld worden', (done) => {
+        const wizard = fixture('vl-wizard-fixture');
+        const panes = wizard.querySelector('[slot="panes"]');
+        wizard.setAttribute('data-vl-next-panes-disabled', '');
+        wizard.setAttribute('data-vl-previous-panes-disabled', '');
+
+        panes.append(document.createElement('vl-wizard-pane'));
+        wizard._panes.forEach((pane) => {
```

Comment:
moet ge hier nog testen dat uwe nieuwe pane bij de wizard._panes zit.

Reply:
Lijkt me in scope van de test inderdaad veiliger om te doen. Ik zal dit toevoegen.